src/main.rs

```diff
@@ -21,6 +22,9 @@ pub enum WallpaperBackend {
 }
 
 fn main() {
```

Comment:
this wont work without

```suggestion
#[tokio::main]
 async fn main() {
```

Reply:
not necessary with the runtime below this, also with how gtk works doing it your way will cause panics